frontend/src/components/NutritionFacts.jsx

```diff
@@ -0,0 +1,141 @@
+import ingredients from '../data/ingredients.json'
+import nutritionData from '../data/nutrition_data.json'
+import './NutritionFacts.css';
+
+export default function NutritionFacts(props) {
+
+    let calories = 0;
+
+    let sumNutrition = {
```

Comment:
Can you move this object inside the `sumNutritionData` and return it from that function instead? There is potential for bugs if this is outside since any double call would potentially double the values.